src/foam/nanos/pm/NullPM.js

```diff
@@ -29,7 +29,16 @@ foam.CLASS({
           type: 'X'
         }
       ],
-      javaCode: '/*nop*/'
+      javaCode: `
```

Comment:
@hchoura what's reason for an implementation?

Reply:
To activate the log.

Reply:
Please explain.  If you need PMs for every create, then change the percentage in the nspec, it's presently set to 1%, change it 100. 

foam/nanos/pm/services.jrl

p({
  class:"foam.nanos.boot.NSpec",
  name:"PM",description:"Configurable PM Factory",
  authenticate:false,
  pm:false,
  service:{
    class:"foam.nanos.pm.PMFactory",
    percentage:1
  }
})